applications/luci-app-vpnbypass/root/etc/uci-defaults/40_luci-vpnbypass

```diff
@@ -1,11 +1,4 @@
 #!/bin/sh
-uci -q batch <<-EOF >/dev/null
```

Comment:
https://github.com/openwrt/packages/blob/5da8ae16b23293250ac6c28bd727101b9c539500/net/vpnbypass/files/vpnbypass.init#L115
vpnbypass doesn't contain a trigger for config `vpnbypass`. Should I keep this?

Reply:
good question.
Maybe we should add the vpnbypass file to the trigger service?

Reply:
Please add a trigger for the `vpnbypass` uci config